nav2_behavior_tree/nav2_tree_nodes.xml

```diff
@@ -283,6 +283,7 @@
     <Condition ID="IsPathValid">
       <input_port name="path"> Path to validate </input_port>
       <input_port name="server_timeout"> Server timeout </input_port>
+      <input_port name="max_cost"> Maximum cost of the path </input_port>
```

Comment:
We need a docs PR to update the configuration guide to include this new port as well :-)